src/createjs/utils/BrowserDetect.js

```diff
@@ -63,7 +63,7 @@ this.createjs = this.createjs||{};
 	 */
 	function BrowserDetect() {
 		throw "BrowserDetect cannot be instantiated";
-	};
```

Comment:
But I'm removing the semicolon not adding. I don't understand what you're saying, but I can changed it.